modules/mauth-signer-http4s/src/main/scala/com/mdsol/mauth/http4s/client/Implicits.scala

```diff
@@ -0,0 +1,46 @@
+package com.mdsol.mauth.http4s.client
+
+import com.mdsol.mauth.models.SignedRequest
+import org.http4s.headers.`Content-Type`
+import org.http4s.{headers, Header, Headers, Method, Request, Uri}
+import org.typelevel.ci.CIString
+
+import scala.annotation.nowarn
+import scala.collection.immutable
+
+object Implicits {
+
+  implicit class NewSignedRequestOps(val signedRequest: SignedRequest) extends AnyVal {
+
+    /** Create an akka-http request from a [[models.SignedRequest]]
+      */
+    def toHttp4sRequest[F[_]]: Request[F] = {
+      val contentType: Option[`Content-Type`] = extractContentTypeFromHeaders(signedRequest.req.headers)
+      val headersWithoutContentType: Map[String, String] = removeContentTypeFromHeaders(signedRequest.req.headers)
+
+      val allHeaders: immutable.Seq[Header.Raw] = (headersWithoutContentType ++ signedRequest.mauthHeaders).toList
+        .map { case (name, value) =>
+          Header.Raw(CIString(name), value)
+        }
+
+      Request[F](
+        method = Method.fromString(signedRequest.req.httpMethod).getOrElse(Method.GET),
+        uri = Uri(path = Uri.Path.unsafeFromString(signedRequest.req.uri.toString)),
```

Comment:
i understand why you'd want to do this, but (especially) in library code we should avoid doing these 2 things: forcing a fallback value and using unsafe methods. The rationale is that debugging potential issues coming from this would be ridiculously difficult. 
I'd claim that this method's signature should be `def toHttp4sRequest[F[_]]: F[Request[F]]`, in order to delegate to the caller the decision on what to do for:
1. whether the method used from within the wrapped request instance is not derivable as canonically named by http4s
2. the URI can't be built based on the supplied String representation